packages/repository/DEVELOPING.md

```diff
@@ -30,8 +30,8 @@ Hope this example shows the idea that even they both use the `base model` from
 Juggler, LB4 model doesn't use, or even have access to some fields/entries in
 the Juggler model!
 
-As mentioned before, we are using the legacy juggler bridge as a temporary
-solution. This may change in the future since the Juggler is pretty.. you know,
-legacy. However, it's important to realize how LB4 deals with models.
+As mentioned before, we are using the juggler bridge as a temporary solution.
```

Comment:
I think we'll need to rephrase this or even remove it altogether. @raymondfeng @bajtos thoughts?

Reply:
+1 in removing this paragraph.

Reply:
Updated, thanks!